### ISSUE TITLE
Release 0.2?

### ISSUE THREAD

**Issue by jproffitt** [mitsuhiko/htmldiff#7]
Is version 0.2 ready to be uploaded to PyPi?

**Comment by hugovk** [comment]
Yes please!

In the meantime, you can install from GitHub:

```bash
pip install https://github.com/edsu/htmldiff/tarball/master#egg=htmldiff-0.2
```

**Comment by edsu** [comment]
The https://github.com/docnow/diffengine project could really use this since `--process-dependency-links` is no longer available in pip. This means people can't `pip install diffengine`.

https://github.com/DocNow/diffengine/issues/57

**Comment by weitzman** [comment]
Yes, I also was unable to easily install diffengine. Please make a new release if possible.

**Comment by mitsuhiko** [comment]
I don't think me making a new release is going to fix much in the longer term. Ideally someone else can take over the library.

**Comment by hugovk** [comment]
Would @edsu be interested in taking over? I could also help with some basic maintenance things. Perhaps also transfer it @DocNow or @Jazzband?

**Comment by edsu** [comment]
@mitsuhiko can you transfer the ownership to @docnow and give us the ability to do PyPI releases? I dont think we will have a huge amount of time to move the library forward but we can make small adjustments as needed. Also, if @jazzband is interested in doing maintenance I would defer to them.

**Comment by edsu** [comment]
@mitsuhiko alternatively I suppose we can just fork it... Although this seems like such a drastic action to have you simply do a push to PyPI of what's already on master doesn't it?

**Comment by edsu** [comment]
Given the inactivity here I think we have no choice but to fork. Since you are unable to upload what is here to PyPI as v0.2 I willl be creating htmldiff2, and renaming my fork appropriately.

**Comment by edsu** [comment]
Ok, I'm about to upload [htmldiff2](https://github.com/edsu/htmldiff2), a friendly fork of htmldiff to PyPI. If it ever needs to be deprecated because this was updated please let me know!
